workflows/bacterial_genomics/quality_and_contamination_control/README.md

```diff
@@ -0,0 +1,27 @@
+# Quality and Contamination control workflow for paired end data (v1.0)
```

Comment:
Is this a general purpose workflow or is any part of this tailored to bacterial genome assembly ?

Reply:
The workflow has a general purpose. Simply use the trimmed fastqs from this workflow to launch “bacterial genome assembly”